### ISSUE TITLE
Add link to #tomatoes channel at freenode

### ISSUE THREAD

**Issue by potomak** [tomatoes-app/tomatoes#269]
The footer should include a link to irc://irc.freenode.net/tomatoes.

A static page could include the IRC web client, see example usage at https://github.com/hackatron/hackatron.github.io/blob/master/about.html.

**Comment by PezCoder** [comment]
@potomak Hey, I want to help out here but not sure how to go about it.
Please correct me if i'm wrong, I need to append a link with [_footer.html.erb#L15](https://github.com/tomatoes-app/tomatoes/blob/master/app/views/shared/_footer.html.erb#L15) which will be displayed at `IRC` & should link to  `irc://irc.freenode.net/tomatoes`

**Comment by potomak** [comment]
@PezCoder thanks for reaching out.

Actually I wanted to put it between line 20 and 21 of the footer, see https://github.com/tomatoes-app/tomatoes/blob/master/app/views/shared/_footer.html.erb#L20-L21.

You could make the link point to `https://webchat.freenode.net/?channels=#tomatoes`, a web IRC client that could be used to join the chat room.

It should look like this:

![image](https://cloud.githubusercontent.com/assets/153842/24084297/f1177c04-0cbd-11e7-80bd-d9e0a6d15028.png)

To summarize you need to:

- [ ] add a new translation `footer.join_chat_html` in the file `config/locales/en.yml`, the value of the translation should include the link, in a similar way `footer.follow_updates_html` does
- [ ] add the new line in the footer between lines 20 and 21

**Comment by potomak** [comment]
Closed by https://github.com/tomatoes-app/tomatoes/pull/285.
